CardPayments/src/main/java/com/paypal/android/cardpayments/CardRequestFactory.kt

```diff
@@ -1,11 +1,11 @@
 package com.paypal.android.cardpayments
 
-import com.paypal.android.cardpayments.api.GetOrderRequest
 import com.paypal.android.corepayments.APIRequest
 import com.paypal.android.corepayments.HttpMethod
+import com.paypal.android.corepayments.api.CoreRequestFactory
 import org.json.JSONObject
 
-internal class CardRequestFactory {
+internal class CardRequestFactory : CoreRequestFactory() {
```

Comment:
🧶 : can we keep `createGetOrderInfoRequest` in this class and rename `CardRequestFactory` to `OrdersRequestFactory` or something? I'm thinking about [this article](https://www.cinqict.nl/blog/dont-repeat-yourself-and-dont-pay-inheritance-tax) and the section on _Inheritance Tax_.

Reply:
Also starting to feel like request factories will become obsolete soon. That design was temporary, there's definitely a better way to keep this logic in the `CardAPI` class.